### DISCUSSION TITLE
GraphQL error on entering workflow

### DISCUSSION THREAD

**Discussion by Lodin** [community/community#25505] [Actions]
<p>Playing with Github Actions, I faced very strange issue. When I try to enter my workflow, it redirects me to the <a href="https://github.com/_graphql/MarkNotificationSubjectAsRead" rel="nofollow noopener">https://github.com/_graphql/MarkNotificationSubjectAsRead</a> page which contains the following message:</p>
<pre><code>Must be requested with X-Requested-With: XMLHttpRequest
</code></pre>
<p>Here is the list of actions I have:</p>
<p><a href="https://github.com/Lodin/test-github-actions/actions" rel="nofollow noopener">https://github.com/Lodin/test-github-actions/actions</a></p>
<p>I also cannot stop executing the workflows (<a href="https://github.community/t/cant-cancel-github-action-workflow/16060">same issue</a>).</p>

**Comment by ibakshay** [comment]
<p>I am also facing the same issue for the past ten minutes or so. </p>

**Comment by dawidd6** [comment]
<p>Same here. All workflows.</p>

**Comment by dreamer** [comment]
<p>Same issue - I can’t access any of my job logs. Workflows UI changed a bit, so I guess GitHub released an update.</p>

**Comment by ethomson** [comment, marked as answer]
<p>Apologies for the inconvenience.  We’re investigating this now.</p>

**Comment by string-areeb** [comment]
<p>Thanks, issue has been fixed for me</p>

**Comment by ibakshay** [comment]
<p>issue is fixed. thank you for being very responsive. </p>

**Comment by ethomson** [comment]
<p>Thanks for letting us know!</p>

**Comment by OlofMoriya** [comment]
<p><strong>I ran into this problem now. Something that needs to be manually fixed?</strong></p>
<p><strong><a href="https://github.com/_graphql/MarkNotificationSubjectAsRead" rel="nofollow noopener">https://github.com/_graphql/MarkNotificationSubjectAsRead</a></strong></p>

**Comment by cice** [comment]
<p>Yes, it seems it is back. Have this issue now too.</p>

**Comment by arg** [comment]
<p>I also get this error message: when I open job logs on iPad (latest iOS version) it redirects me to <a href="https://github.com/_graphql/MarkNotificationSubjectAsRead" rel="noopener nofollow ugc">https://github.com/_graphql/MarkNotificationSubjectAsRead</a></p>
